pkg/k8sutil/deployments.go

```diff
@@ -122,11 +122,11 @@ func (k *K8sutil) CreateClientDeployment(baseImage string, replicas *int32, java
 		limitMemory, _ := resource.ParseQuantity(resources.Limits.Memory)
 		requestCPU, _ := resource.ParseQuantity(resources.Requests.CPU)
 		requestMemory, _ := resource.ParseQuantity(resources.Requests.Memory)
-		scheme := v1.URISchemeHTTP
+//		scheme := v1.URISchemeHTTP
```

Comment:
comments?

Reply:
ommented for testing purpose, Addressed in the latest commit.